test/Microsoft.ML.OnnxTransformTest/Microsoft.ML.OnnxTransformTest.csproj

```diff
@@ -5,10 +5,32 @@
     <ProjectReference Include="..\..\src\Microsoft.ML.ImageAnalytics\Microsoft.ML.ImageAnalytics.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.PipelineInference\Microsoft.ML.PipelineInference.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.ML.OnnxTransform\Microsoft.ML.OnnxTransform.csproj" />
+    <ProjectReference Include="..\..\src\Microsoft.ML.DnnImageFeaturizer.ResNet18\Microsoft.ML.DnnImageFeaturizer.ResNet18.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Predictor.Tests\Microsoft.ML.Predictor.Tests.csproj" />
     <ProjectReference Include="..\Microsoft.ML.TestFramework\Microsoft.ML.TestFramework.csproj" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.ML.Onnx.TestModels" Version="0.0.2-test" />
   </ItemGroup>
+
+  <ItemGroup>
+    <DnnPrepModelFiles Include="$(RepoRoot)bin\obj\DnnImageModels\ResNetPrepOnnx\ResNetPreprocess.onnx" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <DnnMainModelFiles Include="$(RepoRoot)bin\obj\DnnImageModels\ResNet18Onnx\ResNet18.onnx" />
```

Comment:
Why do you split these two files into 2 separate Items?

I think this code would be simpler if you did:

```xml
<ItemGroup>
  <DnnModelFile Include="$(RepoRoot)bin\obj\DnnImageModels\ResNetPrepOnnx\ResNetPreprocess.onnx" />
  <DnnModelFile Include="$(RepoRoot)bin\obj\DnnImageModels\ResNet18Onnx\ResNet18.onnx" />
</ItemGroup>

  <Target Name="CopyModelFiles" AfterTargets="Build">
    <Copy 
      SourceFiles="@(DnnModelFile)"
      DestinationFolder="$(OutDir)\DnnImageModels\ResNet18Onnx" />
  </Target>
```

Reply:
This hasn't been resolved. You can follow the same approach here as you did in `DnnImageFeaturizer.ModelRedist.proj`.

---
In reply to: [234758554](https://github.com/dotnet/machinelearning/pull/1447#discussion_r234758554) [](ancestors = 234758554)

Reply:
Actually, an easier approach would be just to add these two files as `Content` items, set `CopyToOutputDirectory` and `Link` properties as appropriate. Then the build will just copy the files.

---
In reply to: [236764547](https://github.com/dotnet/machinelearning/pull/1447#discussion_r236764547) [](ancestors = 236764547,234758554)

Reply:
You are right, wrote this using it as content files. The issue with the other approach was that the Copy action cannot operate on multiple destination folders at once, and I needed the two models to end up in separate folders.

---
In reply to: [236764961](https://github.com/dotnet/machinelearning/pull/1447#discussion_r236764961) [](ancestors = 236764961,236764547,234758554)